src/otaclient/ota_core.py

```diff
@@ -1195,6 +1199,7 @@ def __init__(
                 failure_reason=f"failed to determine boot controller or create_standby mode: {e!r}",
             )
             return
+        self._metrics.bootcontroller_type = _boot_controller_type.__name__
```

Comment:
About the bootcontroller_type here, I recommend that we add an class attribute `boot_controller_type` to each boot controller implementation, which is assigned with enum in  `_ecu_info.BootloaderType`.

Reply:
Thank you, fixed in https://github.com/tier4/ota-client/pull/611/commits/162e4b692fc798e7cafa6a4f7e19aac2b49fd1dd :+1: